CRITICAL - 2022-04-24 14:02:03 --> Invalid file: LoginController.php
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('LoginController...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('LoginController', Array, true)
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\Home.php(9): view('LoginController')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\Home->index()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-04-24 14:02:27 --> Invalid file: LoginController.php
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('LoginController...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('LoginController', Array, true)
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\Home.php(9): view('LoginController')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\Home->index()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-04-24 14:06:56 --> include(functions.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(functio...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:08:04 --> include(functions.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(functio...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:08:42 --> include(functions.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(functio...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\Home.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\Home->index()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:45:38 --> Call to undefined function registerUser()
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include()
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-24 14:47:07 --> include(RegisterController.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(Registe...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:47:58 --> include(RegisterController.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(Registe...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:48:00 --> include(RegisterController.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(Registe...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:48:09 --> include(RegisterController.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(Registe...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 14:48:31 --> include(RegisterController.php): Failed to open stream: No such file or directory
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(Registe...', 'C:\\Users\\y_eri\\...', 11)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Views\RegisterView.php(11): include()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(210): include('C:\\Users\\y_eri\\...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Common.php(1089): CodeIgniter\View\View->render('RegisterView', Array, true)
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(9): view('RegisterView')
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->register()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-24 16:36:35 --> Allowed fields must be specified for model: App\Models\RegisterModel
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(1138): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\Regi...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(713): CodeIgniter\BaseModel->doProtectFields(Array)
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(22): CodeIgniter\Model->insert(Array)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->saveUser()
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-24 16:39:33 --> Allowed fields must be specified for model: App\Models\RegisterModel
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(1138): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\Regi...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(713): CodeIgniter\BaseModel->doProtectFields(Array)
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(22): CodeIgniter\Model->insert(Array)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->saveUser()
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-24 16:41:58 --> Allowed fields must be specified for model: App\Models\RegisterModel
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(1138): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\Regi...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(713): CodeIgniter\BaseModel->doProtectFields(Array)
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(22): CodeIgniter\Model->insert(Array)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->saveUser()
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-24 16:52:25 --> Allowed fields must be specified for model: App\Models\RegisterModel
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(1138): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\Regi...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(713): CodeIgniter\BaseModel->doProtectFields(Array)
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\RegisterController.php(35): CodeIgniter\Model->insert(Array)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\RegisterController->saveUser()
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-04-24 18:39:39 --> Column 'country' cannot be null
ERROR - 2022-04-24 18:40:00 --> Column 'country' cannot be null
ERROR - 2022-04-24 18:41:57 --> Column 'country' cannot be null
ERROR - 2022-04-24 18:42:23 --> Column 'country' cannot be null
ERROR - 2022-04-24 18:42:28 --> Column 'country' cannot be null
ERROR - 2022-04-24 18:42:28 --> Column 'country' cannot be null
CRITICAL - 2022-04-24 19:42:26 --> Undefined property: App\Controllers\LoginController::$load
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\Users\\y_eri\\...', 13)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 19:48:33 --> Undefined property: App\Controllers\LoginController::$load
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\Users\\y_eri\\...', 13)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 19:50:42 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::get_where()
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getUserLogin('erick.yosue@gma...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 20:41:39 --> Undefined property: App\Controllers\LoginController::$LoginModel
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\Users\\y_eri\\...', 14)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 21:12:11 --> mysqli_result::fetch_object(): Argument #1 ($class) must be a valid class name, App\Models\User given
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\MySQLi\Result.php(148): mysqli_result->fetch_object('App\\Models\\User')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(149): CodeIgniter\Database\MySQLi\Result->fetchObject('App\\Models\\User')
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(106): CodeIgniter\Database\BaseResult->getCustomResultObject('App\\Models\\User')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(377): CodeIgniter\Database\BaseResult->getResult('App\\Models\\User')
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(210): CodeIgniter\Database\BaseResult->getFirstRow('App\\Models\\User')
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(18): App\Models\LoginModel->getUserLogin('email', '')
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-04-24 21:12:39 --> mysqli_result::fetch_object(): Argument #1 ($class) must be a valid class name, App\Models\User given
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\MySQLi\Result.php(148): mysqli_result->fetch_object('App\\Models\\User')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(149): CodeIgniter\Database\MySQLi\Result->fetchObject('App\\Models\\User')
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(106): CodeIgniter\Database\BaseResult->getCustomResultObject('App\\Models\\User')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(377): CodeIgniter\Database\BaseResult->getResult('App\\Models\\User')
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(210): CodeIgniter\Database\BaseResult->getFirstRow('App\\Models\\User')
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(18): App\Models\LoginModel->getUserLogin('email', '')
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-04-24 21:14:12 --> mysqli_result::fetch_object(): Argument #1 ($class) must be a valid class name, App\Models\User given
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\MySQLi\Result.php(148): mysqli_result->fetch_object('App\\Models\\User')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(149): CodeIgniter\Database\MySQLi\Result->fetchObject('App\\Models\\User')
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(106): CodeIgniter\Database\BaseResult->getCustomResultObject('App\\Models\\User')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(377): CodeIgniter\Database\BaseResult->getResult('App\\Models\\User')
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(210): CodeIgniter\Database\BaseResult->getFirstRow('App\\Models\\User')
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(18): App\Models\LoginModel->getUserLogin('email', '')
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-04-24 21:16:41 --> Undefined constant "App\Controllers\RegisterView"
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-04-24 21:21:45 --> mysqli_result::fetch_object(): Argument #1 ($class) must be a valid class name, App\Models\User given
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\MySQLi\Result.php(148): mysqli_result->fetch_object('App\\Models\\User')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(149): CodeIgniter\Database\MySQLi\Result->fetchObject('App\\Models\\User')
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(106): CodeIgniter\Database\BaseResult->getCustomResultObject('App\\Models\\User')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Database\BaseResult.php(377): CodeIgniter\Database\BaseResult->getResult('App\\Models\\User')
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Model.php(210): CodeIgniter\Database\BaseResult->getFirstRow('App\\Models\\User')
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(18): App\Models\LoginModel->getUserLogin('email', 'y_erick.11@hotm...')
#8 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#9 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-04-24 21:28:04 --> Attempt to read property "password" on array
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\Users\\y_eri\\...', 19)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 21:28:13 --> syntax error, unexpected single-quoted string "password", expecting identifier or variable or "{" or "$"
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\Users\\y_eri\\...')
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-04-24 21:29:06 --> Attempt to read property "password" on array
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\Users\\y_eri\\...', 19)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 21:32:50 --> Attempt to read property "password" on array
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\Users\\y_eri\\...', 19)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 21:33:50 --> Array to string conversion
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\Users\\y_eri\\...', 18)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 21:35:07 --> Attempt to read property "password" on array
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\Users\\y_eri\\...', 19)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-24 21:36:57 --> Undefined array key 4
#0 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\app\Controllers\LoginController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\Users\\y_eri\\...', 19)
#1 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(832): App\Controllers\LoginController->authentication()
#2 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#3 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\y_eri\OneDrive\Escritorio\xampp\htdocs\segundoProyecto\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
